controllers/managedosversionchannel_controller.go

```diff
@@ -460,6 +463,12 @@ func (r *ManagedOSVersionChannelReconciler) createSyncerPod(ctx context.Context,
 		return err
 	}
 
+	// If we don't update the LastSyncedTime on Pod creation, we will loop
```

Comment:
We need this to avoid  `reachedNextInterval` at line 181 of this same file to keep being true which would land to a create/delete infinite loop